packages/react-bootstrap-table2/style/react-bootstrap-table.scss

```diff
@@ -22,6 +22,10 @@
     margin: 10px 6.5px;
   }
 
+  th[data-row-selection] {
```

Comment:
any reason that dont use class?

Reply:
I thought it might be more clear to distinguish from what is for user and what is for us. In the future, we may allow user to customize the` class` and it will be easier to implement without any conflicts.